suite-native/module-send/src/components/FiatAmountInput.tsx

```diff
@@ -57,6 +57,7 @@ export const FiatAmountInput = ({
         setValue(cryptoFieldName, converters?.convertFiatToCrypto?.(transformedValue), {
             shouldValidate: true,
         });
+        setValue('setMaxOutputId', undefined);
```

Comment:
The name is also used in `CryptoAmountInput`, would it make sense to create a constant for that?

Reply:
That is the name of the field from [SendOutputsFormValues](https://github.com/trezor/trezor-suite/blob/acb46c296763d940e73bb7e7e33ad917c9ca0ad9/suite-native/module-send/src/sendOutputsFormSchema.ts#L139) and Typescript should tell you what the possible values are.

Unfortunately, the typing of our custom form hook was wrong. So I fixed the typescript type check in [acb46c2](https://github.com/trezor/trezor-suite/pull/14982/commits/acb46c296763d940e73bb7e7e33ad917c9ca0ad9). Since TypeScript tells you what you should fill in, the constant is not needed in my opinion.

Reply:
OK, then it makes sense like this. 👍